<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpsTerrain</class>
 <widget class="QWidget" name="OpsTerrain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>549</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="terrainVerticalLayout_2">
   <item>
    <widget class="QGroupBox" name="TerrainBox">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>150</height>
      </size>
     </property>
     <property name="title">
      <string>Source Options</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <layout class="QVBoxLayout">
        <item>
         <widget class="QCheckBox" name="ShowTerrainB">
          <property name="text">
           <string>Show Terrain</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="terrainFileLayout">
          <item>
           <widget class="QLabel" name="terrainFileLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="whatsThis">
             <string>Specify the terrain file to use.</string>
            </property>
            <property name="text">
             <string>Terrain file:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="kcfg_TerrainSource">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="selectTerrainFileB">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>28</width>
              <height>28</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>28</width>
              <height>28</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="terrainSourceCorrectAzLayout">
          <item>
           <widget class="QLabel" name="terrainSourceCorrectAzLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Rotate terrain in azimuth degrees so north is actually north</string>
            </property>
            <property name="text">
             <string>Terrain image azimuth correction degrees</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="kcfg_TerrainSourceCorrectAz">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Rotate terrain in azimuth degrees so north is actually north</string>
            </property>
            <property name="minimum">
             <number>-360</number>
            </property>
            <property name="maximum">
             <number>360</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>200</width>
              <height>16</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="TerrainSpeedupGroupBox_2">
     <property name="title">
      <string>Speedup options</string>
     </property>
     <layout class="QVBoxLayout" name="TerrainSpeedupVerticalLayout">
      <item>
       <layout class="QHBoxLayout" name="terrainDownsampleLayout">
        <item>
         <widget class="QLabel" name="terrainDownsamplingLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Set the coarseness of the terrain image rendered. 1 is full resolution, but can be slow, 6 or more would be coarse resolution and fast.</string>
          </property>
          <property name="text">
           <string>Terrain Downsampling</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="kcfg_TerrainDownsampling">
          <property name="toolTip">
           <string>Set the coarseness of the terrain image rendered. 1 is full resolution, but can be slow, 6 or more would be coarse resolution and fast.</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>8</number>
          </property>
          <property name="value">
           <number>4</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>200</width>
            <height>16</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="ShowTerrainPanningB">
        <property name="text">
         <string>Show Terrain While Panning</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="TerrainSmoothB">
        <property name="text">
         <string>Smooth Pixels</string>
        </property>
        <property name="toolTip">
         <string>Creates a smoother image at the cost of some extra computation.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="TerrainSkipB">
        <property name="text">
         <string>Skip Pixels Speedup</string>
        </property>
        <property name="toolTip">
         <string>A speedup with minor image quality cost.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="TerrainTransparentB">
        <property name="text">
         <string>Transparency Speedup</string>
        </property>
        <property name="toolTip">
         <string>A speedup with minor image quality cost.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
